src/main/java/me/arminb/sara/controller/UserController.java

```diff
@@ -0,0 +1,77 @@
+package me.arminb.sara.controller;
+
+import me.arminb.sara.entities.User;
+import me.arminb.sara.services.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/user")
```

Comment:
change this to /users. read this on designing REST APIs http://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api